lib/beaker-pe/install/pe_utils.rb

```diff
@@ -507,6 +507,7 @@ def simple_monolithic_install(master, agents, opts={})
           fetch_pe([master], opts)
           prepare_host_installer_options(master)
           generate_installer_conf_file_for(master, [master], opts)
+          step "Install PE on master"
```

Comment:
Sure.  Wasn't sure which convention I should use since both were used in the method already.